_data/toc-dev.yml

```diff
@@ -1,8 +1,41 @@
 
 
-- pagetitle: Blockchain Developers
-
-- title: Developers Guide
+- pagetitle: FIO Developers
+- title: How to Contribute
+  url: "docs/contribute"
+  links:
+    - title: FIO Improvement Proposals
+      url: "docs/contribute/fips"
+    - title: Bug Bounty Program
+      url: "docs/contribute/bounty"
+      children:
+        - title: Bounty Program Scope
+          url: "docs/contribute/bounty-scope"
+        - title: Bounty Program Rules and Guidance
+          url: "docs/contribute/bounty-howto"
+        - title: Bounty Rewards
+          url: "docs/contribute/bounty-rewards"          
+        - title: Bounty Program FAQ
+          url: "docs/contribute/bounty-faq"
+    - title: Participate in Governance
```

Comment:
This participate in governance does not really belong under a "FIO Developers" section since it is pertinent to large token holders (like our wallet and exchange partners) and general users.

My current thinking is that we may want to combine the End User Guide and FIO Developers into a single "FIO Contributors" or "FIO Community" or "Community Contributors" section that has a Dev guide, some user guides, and the How to Contribute section.